src/utils/vcs.rs

```diff
@@ -1585,6 +1590,51 @@ mod tests {
         std::env::remove_var("GITHUB_EVENT_NAME");
     }
 
+    #[test]
+    #[serial(github_env)]
```

Comment:
This is set to serial because we're setting github env vars (like other tests that also do this). We can also split up these tests but I figured it saves all the cleanup and teardown before and after every test if we do it all together.